src/pages/getting-started/index.mdx

```diff
@@ -38,121 +62,106 @@ From a more tech perspective, the LiveChat Platform provides:
 - <a href="https://www.helpdesk.com/help/what-are-teams/" target="_blank">Teams</a>
   
-Read more about HelpDesk <a href="https://www.helpdesk.com/help/what-is-helpdesk/" target="_blank">here</a>, and if you want to start your experience, check out <a href="https://www.helpdesk.com/help/starting-with-helpdesk/" target="_blank">3 simple steps showing how to begin</a>.
+Leran more about HelpDesk <a href="https://www.helpdesk.com/help/what-is-helpdesk/" target="_blank">here</a>, and start your experience with it by checking out <a href="https://www.helpdesk.com/help/starting-with-helpdesk/" target="_blank">3 simple steps showing how to begin</a>.
 
-# What LiveChat can be
+## What HelpDesk can be
 
-We're constantly trying to come forward with the new ways customers could integrate with LiveChat. Certain business needs call for specific customization, whether it's creating an app or just tweaking the look & feel. Currently, you can extend LiveChat in three main areas:
+As the Platform tries to follow the needs of our developers so they can build a wide range of integrations that perfectly fit their customers, you can extend HelpDesk in many different areas.
 
-- [LiveChat user interfaces](#extending-the-livechat-ui)
-- [Platform APIs](#platform-apis)
-- [Interacting with data](#interacting-with-data)
+### Extending HelpDesk
 
-# Extending the LiveChat UI
+The first and definitely most fun way to extend HelpDesk is by playing with its interface. It can be done by adding widgets and placing them in many different locations of your app. You can choose one of the following placements: the Details section, Fullscreen apps, or Settings (<a href="https://developers.livechat.com/docs/extending-agent-app" target="_blank">Learn more...</a>). The <a href="https://developers.livechat.com/docs/extending-agent-app/products-sdk/helpdesk-sdk" target="_blank">HelpDesk SDK</a> is currently the primary way to develop your own widgets.
 
-One of the paths is to extend user interfaces. We provide developers with tools to modify the Agent App, as well as the Chat Widget interface.
+You can also let users sign in to your service via <a href="https://developers.livechat.com/docs/authorization/sign-in-with-livechat#accounts-sdk" target="_blank">Accounts SDK</a> or automate various tasks using [HelpDesk APIs](#helpdesk-api).
 
-## Agent App
+### HelpDesk API
 
-<div style={{ marginBottom: 20 }}>
-  <img
-    alt="LiveChat Chat Details application"
-    src="/images/getting-started/livechat-agent-app-details-widget.png"
-    width="800px"
-    height="467px"
-  />
-</div>
+When you build HelpDesk applications with our Platform, they may need to use our <a href="https://api.helpdesk.com/docs" target="_blank">HelpDesk API</a>. To allow you to play with it and discover its potential, which is developed in many useful areas, HelpDesk API is fully available. You can experiment with:
```

Comment:
I'd look for a synonym of "play with" as it appears a couple of times in this doc